app/models/filters/filter_base.rb

```diff
@@ -536,6 +554,16 @@ def effective_project_ids_from_project_groups
         pluck(p_t[:id].as('project_id'))
     end
 
+    def effective_project_ids_from_secondary_project_groups
+      projects = secondary_project_group_ids.reject(&:blank?).map(&:to_i)
```

Comment:
`secondary_project_group_ids` already rejects blank.  Maybe it should also convert to integers?  Then  we could just reference `secondary_project_group_ids`

Reply:
Hmm -- it already does `map(&:to_i) as well... This is the same as `effective_project_ids_from_project_groups` -- I worry that maybe we do this again so that it comes back right from the db?